pkg/upstream/admin-console.go

```diff
@@ -32,6 +32,29 @@ type UpstreamSettings struct {
 	IncludeMinio           bool
 }
 
+func WriteAdminConsoleFiles(renderDir string, options types.WriteOptions) error {
```

Comment:
i feel like this function doesn't belong in the upstream package IMO, i would just dump this logic in the cli function. but i'll leave the decision up to you.

Reply:
yeah i thought about that but the method that it calls was private to the package. i'm fine with moving it and making that method public though.